### ISSUE TITLE
Fehlender `Zaehlertyp` `WIRBELGASZAEHLER`

### ISSUE THREAD

**Issue by hf-kklein** [bo4e/BO4E-python#577]

**Comment by snsttr** [comment]
@hf-kklein hast du die konkrete Anforderung, dass der Zähler gebraucht wird? Wir gehen bisher davon aus, dass der in der Praxis eigentlich nicht vorkommt.

**Comment by hf-kklein** [comment]
Ja, ich habe die Anforderung. Wieso sollte er in der Praxis nicht vorkommen?
In der Mako wird er als `WGZ` kommuniziert, von daher ist er für mich praxisrelevant.

**Comment by hf-krechan** [comment]
done in #630
